src/js/components/DataChart/index.d.ts

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { 
+  A11yTitleType,
+  GapType,
+  GridAreaType,
+  MarginType,
+} from "../../utils";
+import { ChartProps } from "../Chart";
+import { GridProps } from "../Grid";
+
+type ChartType = {
+  key?: string;
+  keys?: { key: string, color?: ChartProps["color"] }[];
+  a11yTitle?: A11yTitleType;
+  bounds?: ChartProps["bounds"];
+  color?: ChartProps["color"];
+  dash?: ChartProps["dash"];
+  onClick?: ChartProps["onClick"];
+  onHover?: ChartProps["onHover"];
+  overflow?: ChartProps["overflow"];
+  round?: ChartProps["round"];
+  thickness?: ChartProps["thickness"];
+  type?: ChartProps["type"];
+}
+
+export interface DataChartProps {
+  a11yTitle?: A11yTitleType;
```

Comment:
It doesn't appear that `a11yTitle` is supported in DataChart according to the code. From a quick look, I'm not sure Chart is supporting it either (line #14)

Reply:
Added to both.